android/src/main/java/ie/gov/tracing/nearby/ProvideDiagnosisKeysWorker.java

```diff
@@ -80,7 +80,6 @@ public ProvideDiagnosisKeysWorker(@NonNull Context context,
     secureRandom = new SecureRandom();
     repository = new ExposureNotificationRepository(context);
     this.context = context;
-    setForegroundAsync(createForegroundInfo());
```

Comment:
For additional context, an engineer from Google recommended that `setForegroundAsync` should only be called inside `startWork` and shouldn't be in the constructor.